### PR TITLE
Add gpu information

### PR DESCRIPTION
**Pull request by enolfc**
# Summary 

Add information about GPUS.

----

**Related issue :**

### CODE REVIEW

File: source/gpgpu.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.. toctree::
+
+GPGPUs on EGI Cloud
+===================
+
+GPGPUs resources are available on selected providers of the EGI Cloud. These 
+are available as specific **flavors** that when used to instantiate a Virtual
+Machine will make the hardware available for the user.
```
Comment (gwarf) [inline source/gpgpu.rst]:
```suggestion
Machine will make the hardware available to the user.
```

Comment (gwarf) [reply source/gpgpu.rst]: https://forum.wordreference.com/threads/available-to-available-for.2132015/#post-10671203

**Comment by enolfc** [comment]
Sites should provide their ok with this kind of info

File: source/gpgpu.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
++-------------------+---------------------------------+--------------------------------------------+
+| **IISAS-Nebula**  | up to 2 NVIDIA Tesla K20m       | Sponsored access for limited testing,      |
+|                   |                                 | conditions to be negotiated for long-term  |
+|                   |                                 | usage                                      |
++-------------------+---------------------------------+--------------------------------------------+
+| **IFCA-LCG2**     | up to 4 Nvidia GT200GL          | Pay-per-use                                |
+|                   +---------------------------------+                                            |
+|                   | up to 1 Nvidia Titan X          |                                            |
+|                   +---------------------------------+                                            |
+|                   | up to 10 Nvidia Titan 1080ti    |                                            |
```
Comment (gwarf) [inline source/gpgpu.rst]: Not sure Nvidia Titan 1080 Ti exist, isn't it common GeForce like https://www.nvidia.com/en-us/geforce/products/10series/geforce-gtx-1080-ti/ ?

Comment (enolfc) [reply source/gpgpu.rst]: got the information directly from provider, so didn't really check. Also I'm realising capitalisation of NVIDIA is wrong.

Comment (enolfc) [reply source/gpgpu.rst]: Should be fixed now.
Still the approval of the PR depends mostly on the providers giving green light. Contacted them via email

**Comment by gwarf** [comment]
@enolfc I made some comments.

**Comment by enolfc** [comment]
@orviz @tdviet @mariojmdavid  could you take a look?

**Comment by enolfc** [comment]
@arax, CESNET information is also missing. Will you provide GPUs on your new resources?

**Comment by arax** [comment]
@enolfc GPU resources will be added in ~ 1 - 2 months, I currently cannot provide any more information.

**Comment by enolfc** [comment]
> @enolfc GPU resources will be added in ~ 1 - 2 months, I currently cannot provide any more information.

Thanks @arax, then we can make a new PR to cover that

**Comment by mariojmdavid** [comment]
the new openstack rocky (will be stein) is under deployment, we will have 2-3 GPUs, but we are still some time to it
the current openstack infra has 3 GPUs that have been in use for several months and can't commit at the moment

**Comment by enolfc** [comment]
Thanks @mariojmdavid! then I'm removing the reference to your resources and we can add it in a following PR

**Comment by alvarolopez** [comment]
From IFCA the information is correct.

**Comment by enolfc** [comment]
Thanks @alvarolopez!

**Comment by tdviet** [comment]
IISAS-GPU -> IISAS-GPUCloud
Otherwise the information about VM configuration and access condition is correct

**Comment by enolfc** [comment]
Thanks @tdviet, the name of the site is fixed
@gwarf, could you do a quick final review to merge?

**Review by gwarf — APPROVED** [review]
LGTM, thanks!

**Pull request merged**